docs/source/nodes_and_pipelines/pipeline_registry.md

```diff
@@ -0,0 +1,73 @@
+# The pipeline registry
+
+Projects generated using Kedro 0.17.2 or later define their pipelines in `src/<package_name>/pipeline_registry.py`. The `pipeline_registry` module must contain a top-level `register_pipelines()` function that return a mapping from pipeline names to [`Pipeline`](/kedro.pipeline.Pipeline) objects. For example, the pipeline registry in the [Kedro starter for the completed spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) could define the following `register_pipelines()` function that exposes the data processing pipeline, the data science pipeline, and a third, default pipeline that combines both of the aforementioned pipelines:
+
+```python
+def register_pipelines() -> Dict[str, Pipeline]:
+    """Register the project's pipelines.
+
+    Returns:
+        A mapping from pipeline names to ``Pipeline`` objects.
+    """
+    data_processing_pipeline = dp.create_pipeline()
+    data_science_pipeline = ds.create_pipeline()
+
+    return {
+        "__default__": data_processing_pipeline + data_science_pipeline,
+        "data_processing": data_processing_pipeline,
+        "data_science": data_science_pipeline,
+    }
+```
+
+```{note}
+The order in which you add the pipelines together is not significant (`data_science_pipeline + data_processing_pipeline` would produce the same result), since Kedro automatically detects the data-centric execution order for all the nodes in the resulting pipeline.
+```
+
+## Pipeline autodiscovery
+
+In the above example, you need to update the `register_pipelines()` function whenever you create a pipeline that should be returned as part of the project's pipelines. Since Kedro 0.18.3, you can achieve the same result with less code using `find_pipelines()`[/kedro.framework.project.find_pipelines]`:
```

Comment:
Should this be a link: `find_pipelines()`[/kedro.framework.project.find_pipelines]` ? Or just formatted?

Reply:
Ah, oops, nice catch!
```suggestion
In the above example, you need to update the `register_pipelines()` function whenever you create a pipeline that should be returned as part of the project's pipelines. Since Kedro 0.18.3, you can achieve the same result with less code using `find_pipelines()`[/kedro.framework.project.find_pipelines]:
```